### PR TITLE
Added Sidekick in the Monitoring section

### PR DESCRIPTION
**Pull request by ismailegilmez4**

### CODE REVIEW

**Review by mashb1t — COMMENTED** [review]
Hey @ismailegilmez4, first of all i'd i'd like to thank you for your contribution.
I'm not sure if Sidekick really belongs in the docker awesome list as it's not specifically developed with docker as a main usecase but rather a tool based on docker.
Let's wait for the opinion of other maintainers.

**Comment by veggiemonk** [comment]
I haven't seen any interaction with containers while looking through the documentation.
@ismailegilmez4 could you explain to us how this relates to docker containers ?

**Comment by boroskoyo** [comment]
hi @veggiemonk, Sidekick allows you to get real time data from your running applications. It is pretty useful for your containerized applications as it lets you put non breaking breakpoints and logpoints to your apps without stopping & redeploying. 

You can see an example docker config here: https://docs.runsidekick.com/installation/agents/python/installation-1

**Comment by veggiemonk** [comment]
@boroskoyo Thanks for the explanation.

From what you've explained, it looks more related to developing application.
Also, it seems like a paying services since it requires an API key to start.
The documentation and the readme of the product is a bit confusing.

I would be ok to merge it if you add 💲  (heavy_dollar_sign) next to it.

File: README.md
Comment (veggiemonk) [inline README.md:218]:
```suggestion
-   [Sidekick](https://github.com/runsidekick/sidekick) 💲 - Open source live application debugger like Chrome DevTools for your backend. Collect traces and generate logs on-demand without stopping & redeploying your applications.
```

**Review by veggiemonk — APPROVED** [review]

**Pull request merged**
